pace-util/pace/util/quantity.py

```diff
@@ -259,6 +259,7 @@ def __init__(
         origin: Sequence[int] = None,
         extent: Sequence[int] = None,
         gt4py_backend: Union[str, None] = None,
+        mask: Optional[Tuple[bool, bool, bool]] = None,
```

Comment:
I think we should be able to programmatically generate the mask, without taking it as an input. Something like
```
mask = [
    any(dim in coord_dims for dim in quantity.dims)
    for coord_dims in X_DIMS, Y_DIMS, Z_DIMS
]
```

Reply:
You may want to confirm this actually works with someone on the backend (@jdahm?) - my understanding is that mask is specifically for x, y, and z dimensions in that order, always.

Reply:
It would be a big help if you could add unit tests to pace-util making sure 2D storages continue to work.

Reply:
@mcgibbon Yeah, I think we should be able to automatically generate those from a quantity like you wrote.

Reply:
This is now updated to detect the mask inside Quantity.